### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by mroxso**
This dockerizes bostr.

TODO:
- [ ] check if docker container exits properly

### CODE REVIEW

File: compose.yaml
Comment (Yonle) [inline compose.yaml:1]: These comments are not mandatory.
I could remove these later.

**Comment by Yonle** [comment]
You guys love to docker-ing everything huh 🙂
But that's great anyways. Merging

File: compose.yaml
Comment (mroxso) [reply compose.yaml:1]: ah true. forgot to remove them

**Comment by Yonle** [comment]
(you may need to remove `bostr` package in your fork)

**Pull request merged**